jflex/src/main/java/jflex/core/unicode/UnicodeProperties.java

```diff
@@ -19,19 +19,14 @@
 import jflex.chars.Interval;
 import jflex.core.IntCharSet;
 
-/**
- * This class was automatically generated by jflex-unicode-maven-plugin based on data files
- * downloaded from unicode.org.
- *
- * @author JFlex contributors.
- */
+// DO NOT EDIT
+// This class was automatically generated by jflex-unicode-maven-plugin
+// based on data files downloaded from unicode.org.
 public class UnicodeProperties {
 
-  /** Constant {@code UNICODE_VERSIONS="1.1, 1.1.5, 2, 2.0, 2.0.14, 2.1, 2.1.9,"{trunked}} */
```

Comment:
@regisd : how are these javadoc comments generated?  I don't know how to put them back.